internal/app/issue/issue_handler_events.go

```diff
@@ -170,6 +171,9 @@ func createIssueMatches(
 
 		// Create new issue match
 		issue_match := &entity.IssueMatch{
+			Metadata: entity.Metadata{
+				CreatedBy: 1,
+			},
```

Comment:
Don't we want to use the `common.GetCurrentUserId()` function here?

Same for the UserId?

@michalkrzyz can you please check?

Reply:
Yes this is what we should do.